### PR TITLE
Adding a first version of MetaShARK GxIT to generate metadata from raw data

### PR DESCRIPTION
**Pull request by yvanlebras**
(Please replace this header with a description of your pull request. Please include *BOTH* what you did and why you made the changes. The "why" may simply be citing a relevant Galaxy issue.)
(If fixing a bug, please add any relevant error or traceback)
(For UI components, it is recommended to include screenshots or screencasts)

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

File: tools/interactive/interactivetool_metashark.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    <entry_points>
+        <entry_point name="MetaShARK visualisation" requires_domain="True">
+            <port>3838</port>
+            <url>/</url>
+        </entry_point>
+    </entry_points>
+    <command><![CDATA[
+
+mkdir -p /srv/shiny-server/data/ &&
+cp '$infile' /srv/shiny-server/data/inputdata.txt &&
```
Comment (bgruening) [inline tools/interactive/interactivetool_metashark.xml]: @yvanlebras you are copying here only one file, but you can put in multiple files in the UI

Comment (yvanlebras) [reply tools/interactive/interactivetool_metashark.xml]: True! Thank you, I was updating the param to allow multiple files...and forget to change the command line!

Comment (yvanlebras) [reply tools/interactive/interactivetool_metashark.xml]: I tried to add something looking at https://github.com/yvanlebras/galaxy/blob/dev/tools/interactive/interactivetool_jupyter_notebook.xml#L24 . Can you say to me if this is good, and even better to use ln -sf than a cp ?

**Comment by mvdbeek** [comment]
I assume this was merged as part of https://github.com/galaxyproject/galaxy/pull/13811. Thanks @yvanlebras!

**Pull request closed without merging**